scripts/kotlin-lint-check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
```

Comment:
I've been exploring this pre push hook for ktlint 

if we run `./gradlew ktlintcheck` instead of ktlint it provides references to the particular line of code where we need to change 
wouldn't this be better
![image](https://user-images.githubusercontent.com/54740946/101949901-5e123580-3c1a-11eb-8971-11e632b273c6.png)

hoping this  doesn't have any issues with bazel

Reply:
Ktlint Gradle - https://github.com/jlleitschuh/ktlint-gradle

I hope using this doesn't require the user to setup ktlint seperately

Reply:
We are on the migration from gradle to bazel build system, so we will be removing gradle soon, that's the reason we are keeping it gradle free.